threeML/catalogs/Fermi.py

```diff
@@ -605,7 +605,7 @@ def _get_point_source_from_3fgl(fgl_name, catalog_entry, fix=False):
         a = float(catalog_entry["plec_exp_factor"])
         E0 = float(catalog_entry["pivot_energy"])
         b = float(catalog_entry["plec_exp_index"])
-        conv = math.exp(a * E0 ** b)
+        conv = np.exp(a * E0 ** b)
```

Comment:
though it is a small detail, the for non-vendor stuff, the math library is MUCH faster than numpy as it is directly bound to the math core:
```python
In [4]: %timeit np.exp(10.)
548 ns ± 3.26 ns per loop (mean ± std. dev. of 7 runs, 1000000 loops each)

In [5]: %timeit math.exp(10.)
86.3 ns ± 1.03 ns per loop (mean ± std. dev. of 7 runs, 10000000 loops each)
```

Reply:
Thanks, I didn't know that!

(It doesn't make a big difference here since that function would typically be called only once at the beginning of the analysis, to convert a Fermi-LAT catalog to an astromodels model. But I did change it. )